MandelaK/SendIT-API
# #[161853118] write tests for our endpoints

#### What does this PR do?
Write failing tests for five endpoints
#### Description of Task to be completed?
We are supposed to have failing tests as part of the TDD workflow. Because we have not implemented anything yet, we expect the tests to fail, but then we should make the tests pass so that we can know our app works as intended.

#### How should this be manually tested?
This can be tested by cloning the repository, installing all dependencies in requirements.txt  into a virtual environment and running pytest. You will notice that five tests run and all five of them fail. 

#### Any background context you want to provide?
Writing tests before creating solutions is a hallmark of TDD and it helps us write and refactor code with very little chance of reintroducing bugs undetected as we improve the code. 

#### What are the relevant pivotal tracker stories?
#161853118
